lhotse/dataset/sampling/round_robin.py

```diff
@@ -30,7 +31,13 @@ class RoundRobinSampler(CutSampler):
         ...     pass  # profit
     """
 
-    def __init__(self, *samplers: CutSampler, stop_early: bool = False) -> None:
+    def __init__(
+        self,
+        *samplers: CutSampler,
+        stop_early: bool = False,
+        randomize: bool = False,
```

Comment:
I think it'd be better to have an optional list of weights here instead (that has to be the same length as the list of samplers) so that we can control preference for a specific sampler; it could also be `randomize: Union[bool, List[float]] = False` (where True sets uniform weights) but I am OK to merge as-is, up to you.

Reply:
Okay, I'll provide that option too.

Reply:
Added. Please approve the PR when you have some time :)